packages/firestore/test/unit/specs/existence_filter_spec.test.ts

```diff
@@ -82,6 +82,48 @@ describeSpec('Existence Filters:', [], () => {
     );
   });
 
+  specTest('Existence filter uses current state of query', [], () => {
```

Comment:
This test case needs a better name--it's actually testing the positive case where existence filters successfully sync.

Reply:
I split this into two test cases and gave them better names:

- "Existence filter match"
- "Existence filter match with pending update"

Note that we did not have any test cases with existence filter matches before.